sdk/loadtesting/playwright/src/utils/playwrightServiceApicall.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { getTestRunApiUrl } from "./utils.js";
+import { HttpService } from "../common/httpService.js";
+import { TestRunCreatePayload } from "../common/types.js";
+
+/**
+ * Makes a PATCH call to the Playwright Service Test Run API to create or update a test run.
+ *
+ * @param payload - The request payload (displayName, config, ciConfig, etc.).
+ * @returns The parsed JSON response from the API.
+ * @throws If the API call fails (non-2xx response).
+ */
+export class PlaywrightServiceApiCall {
+  private httpService: HttpService;
+
+  constructor(httpService?: HttpService) {
+    this.httpService = httpService ?? new HttpService();
+  }
+
+  async patchTestRunAPI(payload: TestRunCreatePayload): Promise<any> {
+    const baseUrl = getTestRunApiUrl();
+    const apiVersion = "2024-09-01-preview";
+    const token = process.env["PLAYWRIGHT_SERVICE_ACCESS_TOKEN"];
```

Comment:
Use https://github.com/Azure/azure-sdk-for-js/blob/010a873a83ce0906d4a221a9aa76fe3ee80e7993/sdk/loadtesting/playwright/src/utils/utils.ts#L48C14-L48C17